### PR TITLE
referencing 7 days as the default

### PR DESCRIPTION
**Pull request by seanturner**
Addresses GENART "default" comment.

### CODE REVIEW

**Review by jsalowey — COMMENTED** [review]
I'm a little uneasy with this change.  I think the intent is that 7 days is the maximum unless there is an "application profile standard" that changes it.  using "default" makes it sound like this limit is a parameter for any use.   The document already uses the term default in section 4.1.3 so maybe I am picking a nit here.  
I'm tempted to replace default with "unless modified by an application profile standard", but I could also live with "default" but I think we will have implementations that then make this a parameter and set it to 1 month or 1 year.

File: draft-ietf-tls-subcerts.md [hunk 1]
```diff
@@ -372,7 +372,8 @@ valid_time:
 : Time, in seconds relative to the delegation certificate's
   notBefore value, after which the delegated credential is no longer valid.
   Endpoints will reject delegated credentials that expire more than 7 days
-  from the current time (as described in {{client-and-server-behavior}}).
+  from the current time (as described in {{client-and-server-behavior}})
+  based on the default (see {{solution-overview}}.
```
Comment (seanturner) [inline draft-ietf-tls-subcerts.md:376]: I think the answers the mail
```suggestion
from the current time {{client-and-server-behavior}} unless a different value is specified in an application profile standard ({{solution-overview}}).
```

File: draft-ietf-tls-subcerts.md [hunk 2]
```diff
@@ -647,7 +648,7 @@ may result in resuming connections for which the DC has expired.
 
 ## Privacy Considerations
 
-Delegated credentials can be valid for 7 days and it is much easier for a
+Delegated credentials can be valid for 7 days (by default) and it is much easier for a
```
Comment (seanturner) [inline draft-ietf-tls-subcerts.md:651]:
```suggestion
Delegated credentials can be valid for 7 days (in the absence of an application protocol) and it is much easier for a
```

File: draft-ietf-tls-subcerts.md [hunk 3]
```diff
@@ -673,7 +674,8 @@ For (D)TLS 1.2 servers that support RSA key exchange using a DC-enabled end-enti
 certificate, a hypothetical signature forgery attack would allow forging a
 signature over a delegated credential.
 The forged delegated credential could then be used by the attacker as the equivalent of a
-man-in-the-middle certificate, valid for a maximum of 7 days.
+man-in-the-middle certificate, valid for a maximum of 7 days (if the default
+valid_time is used).
```
Comment (seanturner) [inline draft-ietf-tls-subcerts.md:678]:
```suggestion
man-in-the-middle certificate, valid for a maximum of 7 days (in the absence of an application protocol).
```

**Review by jsalowey — APPROVED** [review]
Looks good

**Pull request merged**